integration/helpers/instance.go

```diff
@@ -345,7 +346,7 @@ func NewInstance(cfg InstanceConfig) *TeleInstance {
 		Cert:       cert,
 		TLSCACert:  tlsCACert,
 		TLSCert:    tlsCert,
-		TunnelAddr: net.JoinHostPort(cfg.NodeName, i.GetPortReverseTunnel()),
+		TunnelAddr: i.ReverseTunnel, // TODO(tcsc): net.JoinHostPort(cfg.NodeName, i.GetPortReverseTunnel()),
```

Comment:
Is this TODO item still relevant? I would expand it if so, right now it's not clear what the actual TODO here is.

Reply:
Not relevant any more. Gone.